spec/controllers/course/assessment/assessments_controller_spec.rb

```diff
@@ -265,5 +265,73 @@
         end
       end
     end
+
+    describe '#live_feedback_settings' do
+      render_views
+      let(:assessment) { create(:assessment, course: course) }
+      let(:lang_valid_for_codaveri) { Coursemology::Polyglot::Language::Python::Python3Point12.instance }
```

Comment:
Leave a TODO that this should be updated when `codaveri_language_whitelist` changes in `coursemology2/app/services/codaveri_async_api_service.rb`. Eventually we will have a dynamic list pulled from Codaveri during production, and mock a list during tests.

Reply:
done